output.tf

```
@@ -28,7 +28,7 @@ output "policy_id" {
   description = "The policy ID"
 }
 
-output "switchrole_url" {
-  description = "URL to the IAM console to switch to the role"
-  value       = "${local.enabled ? format(var.switchrole_url, var.member_account_id, var.role_name, module.label.id) : ""}"
+output "switchrole_urls" {
+  description = "List of URL to the IAM console to switch to the roles"
+  value       = ["${formatlist(var.switchrole_url, null_resource.role.*.triggers.account_id, null_resource.role.*.triggers.role_name, null_resource.role.*.triggers.alias)}"]
```

```suggestion
  value       = ["${formatlist(var.switchrole_url_template, null_resource.role.*.triggers.account_id, null_resource.role.*.triggers.role_name, null_resource.role.*.triggers.alias)}"]
```

This is already a template